server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -434,7 +435,7 @@ public String invalidMessage(String argument) {
       case TABLE_BULK_IMPORT: {
         TableOperation tableOp = TableOperation.BULK_IMPORT;
         validateArgumentCount(arguments, tableOp, 4);
-        String tableName = validateTableNameArgument(arguments.get(0), tableOp, NOT_SYSTEM);
+        String tableName = validateNewTableNameArgument(arguments.get(0), tableOp, NOT_SYSTEM);
```

Comment:
This operation is not creating a new table, so it should not do this check.

Reply:
@keith-turner Is the same case for TABLE_IMPORT? Again, in this case, I thought that ctubbsii wanted the limit check applied during imports, as well.

Reply:
table import is creating a new table in the system.

Reply:
@cradal Although similarly named "bulk import" (aka "import directory") is when a new RFile is added to an existing table, whereas "import table" creates a new table from a previously exported table.